app/api_topologies.go

```diff
@@ -24,33 +24,69 @@ var (
 			renderer: render.PodRenderer,
 			Name:     "Pods",
 			Rank:     3,
-			Options: map[string][]APITopologyOption{"system": {
-				{"show", "System pods shown", false, render.FilterNoop},
-				{"hide", "System pods hidden", true, render.FilterSystem},
-			}},
+			Options: []APITopologyOptionGroup{
+				{
+					ID:      "system",
+					Default: "application",
+					Options: []APITopologyOption{
+						{"system", "System pods", render.FilterApplication},
+						{"application", "Application pods", render.FilterSystem},
+						{"both", "Both", render.FilterNoop},
```

Comment:
They get rendered in that order; @davkal should be able to provide some screenshots.